tests/test_charm.py

```diff
@@ -1,15 +0,0 @@
-# Copyright 2021 Canonical Ltd.
```

Comment:
I would have left the unit tests in the same directory since `ops.harness.testing` make supports automatic detection of metadata.yaml from a parent directory.

Reply:
I was not aware of that. Dividing into unit, integration subdirs is common practice in all the references I used.
Could you please post a link to some example code with metadata.yaml relpath expectation?

Reply:
Not sure what example you are looking for but here is the `ops` code https://github.com/canonical/operator/blob/master/ops/testing.py#L239

Reply:
Just to be clear I was only referring to moving unit tests into `tests/unit`. It does not matter where you put integration tests under `tests`.

Reply:
thanks! I think it should be ok because in `inspect.getfile(self._charm_cls)` the argument is the charm, which is "always" in `src/.`